packages/plugins/users-permissions/server/content-types/user/index.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
       minLength: 6,
       configurable: false,
       private: true,
+      required: true,
```

Comment:
Why is it required? If the user authenticates with a provider (facebook for instance), the user would not have any password no?

Reply:
Nice catch I'll only keep the u&p update to avoid breaking stuff